pcsx2/USB/usb-pad/usb-pad.h

```diff
@@ -59,6 +59,15 @@ namespace usb_pad
 		static const TCHAR* LongAPIName(const std::string& name);
 		static int Configure(int port, const std::string& api, void* data);
 		static int Freeze(int mode, USBDevice* dev, void* data);
+		static std::vector<std::string> SubTypes()
+		{
+			std::vector<std::string> ret;
```

Comment:
This probably could just be a initializer list: `{ "blahh", }` etc

Reply:
done